Lib/operator.py

```diff
@@ -221,6 +221,12 @@ def length_hint(obj, default=0):
         raise ValueError(msg)
     return val
 
+# Other Operations ************************************************************#
+
+def call(obj, *args, **kwargs):
```

Comment:
I think you need to make `obj` positional-only. Otherwise a call like `call(dict, obj=3)` will fail with `TypeError: call() got multiple values for argument 'obj'`.

Reply:
sure, done.

Reply:
Thanks!  The user experience isn’t good for reviewers when history is rewritten in a PR, so please do regular commits and push for future PRs (see https://devguide.python.org/pullrequest/#submitting)